src/redux/features/admin/activateUserAcountSlice.ts

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable prettier/prettier */
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+
+import API from '@/api/api';
+import isAuth from '@/helpers/isAuth';
+
+export const activateUserAcount = createAsyncThunk(
+  'activate/user',
+  async (data: string) => {
```

Comment:
Same as here; rename the function parameter to `email` to better reflect its purpose.